server/lib/cleanup/languages/ko.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  sortSentences,
+  clean,
+};
+
+function sortSentences(sentences) {
+  return sentences.sort();
+}
+
+function clean(sentences) {
+  return sentences.map((sentence) => {
+    // Normalization in Korean letters are important,
+    // since there are many kinds of Unicode Character which are same letter.
+    return sentence.normalize('NFC');
```

Comment:
I can't really comment on the normalization per se, I think @ftyers can provide more information here. Is this something that indeed would be valuable to  do?

I have one comment here though: the cleanup script is run when exporting to Common Voice. This means that the sentences within the Sentence Collector would not be normalized. Might this lead to confusion? Also given that the validation would be done on the normalized string instead of what will actually be shown in the Sentence Collector GUI?